core/src/dird/migrate.cc

```diff
@@ -1682,18 +1666,26 @@ void MigrationCleanup(JobControlRecord* jcr, int TermCode)
   PoolMem query(PM_MESSAGE);
 
   Dmsg2(100, "Enter migrate_cleanup %d %c\n", TermCode, TermCode);
-  UpdateJobEnd(jcr, TermCode);
+
```

Comment:
Why was UpdateJobEnd moved down?

Reply:
it is technically the same position, just moved into the `if` statement. 
It used to be executed for both jobs. It now gets executed only for the new migrated job, while the control one gets deleted in the `else` statement, so no need for it to be updated.